app/commands/v2/publish.rb

```diff
@@ -171,19 +174,29 @@ def update_dependencies?
         EditionDiff.new(edition).field_diff.present?
       end
 
-      def send_downstream(content_id, locale, update_type)
-        return unless downstream
-
+      def send_downstream_live
         queue = update_type == 'republish' ? DownstreamLiveWorker::LOW_QUEUE : DownstreamLiveWorker::HIGH_QUEUE
-
         DownstreamLiveWorker.perform_async_in_queue(
           queue,
+          worker_params
```

Comment:
The intent here is to not update dependencies through the DownstreamLiveWorker, and instead update them through the DownstreamDraftWorker. I believe the decision to do it this way was made because the DownstreamDraftWorker resolves dependencies for both content stores, whereas the DownstreamLiveWorker only resolves dependencies for the live content store.

However, thinking about this again, I think I see the problem. Dependency resolution for the draft content store would have happened when the put content request was made, so its unnecessary to do it a 2nd time upon publish. Therefore the better way is as you say to keep the DownstreamDraftWorker doing the dependency resolution.